corehq/apps/accounting/static/accounting/js/pricing_table.js

```diff
@@ -1,30 +1,70 @@
-hqDefine('accounting/js/pricing_table', function () {
-    var pricingTableModel = function (editions, current_edition, isRenewal) {
+hqDefine('accounting/js/pricing_table', [
+    'jquery',
+    'knockout',
+    'underscore',
+    'hqwebapp/js/initial_page_data',
+    "hqwebapp/js/assert_properties",
+], function (
+    $,
+    ko,
+    _,
+    initialPageData,
+    assertProperties
+) {
+    var PricingTable = function (options) {
+        assertProperties.assert(options, [
+            'editions',
+            'planOptions',
+            'currentPlan',
+        ]);
+
         'use strict';
-        var self = {};
+        var self = this;
 
-        self.currentEdition = current_edition;
-        self.isRenewal = isRenewal;
-        self.editions = ko.observableArray(_.map(editions, function (edition) {
-            return pricingTableEditionModel(edition, self.currentEdition);
-        }));
+        self.currentPlan = ko.observable(options.currentPlan);
+        self.editions = options.editions;
 
-        self.selected_edition = ko.observable(isRenewal ? current_edition : false);
-        self.isSubmitVisible = ko.computed(function () {
-            if (isRenewal){
-                return true;
+        self.selectedPlan = ko.observable(options.currentPlan);
+
+        self.showMonthlyPricing = ko.observable(false);
+
+        self.refundCss = ko.computed(function () {
```

Comment:
Just in case you haven't run across the different variations of the `css` binding: you could delete this observable and instead rewrite the css binding as `css: {'hide-refund': showMonthlyPricing}`. I don't think either way is obviously better, just a stylistic difference.

Reply:
Thanks for the advice! I think this is a change that @biyeun added, so I'll let her decide